airflow/contrib/operators/gcp_transfer_operator.py

```diff
@@ -0,0 +1,886 @@
+        Amazon Web Service.
+    :type aws_conn_id: str
+    :param gcp_conn_id: The connection ID used to connect to Google Cloud
+        Platform.
+    :type gcp_conn_id: str
+    :param api_version: API version used (e.g. v1).
+    :type api_version: str
+    :return: Google Storage Transfer service object
+    :rtype: dict
+    """
+    # [START gcp_transfer_job_create_template_fields]
+    template_fields = ('body', )
+    # [END gcp_transfer_job_create_template_fields]
+
+    @apply_defaults
+    def __init__(self,
+                 body,
+                 aws_conn_id='aws_default',
+                 gcp_conn_id='google_cloud_default',
+                 api_version='v1',
+                 *args,
+                 **kwargs):
+        super(GcpTransferServiceJobCreateOperator, self)\
+            .__init__(*args, **kwargs)
+        self.body = deepcopy(body)
+        self.aws_conn_id = aws_conn_id
+        self.gcp_conn_id = gcp_conn_id
+        self.api_version = api_version
+        self._validate_inputs()
+
+    def _validate_inputs(self):
+        TransferJobValidator(
+            body=self.body,
+        ).validate_body()
+
+    @convert_http_exception("Creating transfer job")
+    def execute(self, context):
+        TransferJobPreprocessor(
+            body=self.body,
+            aws_conn_id=self.aws_conn_id,
+        ).process_body()
+        hook = GCPTransferServiceHook(
+            api_version=self.api_version,
+            gcp_conn_id=self.gcp_conn_id,
+        )
+        return hook.create_transfer_job(
+            body=self.body
+        )
+
+
+class GcpTransferServiceJobUpdateOperator(BaseOperator):
+    """
+    Updates a transfer job that runs periodically.
+
+    .. seealso::
+        For more information on how to use this operator, take a look at the guide:
+        :ref:`howto/operator:GcpTransferServiceJobUpdateOperator`
+
+    :param job_name: Name of the job to be updated
+    :type job_name: str
+    :param body: The request body, as described in
+        https://cloud.google.com/storage-transfer/docs/reference/rest/v1/transferJobs/patch#request-body
+        With three additional improvements:
+
+        * dates can be given in the form datetime.date
+        * times can be given in the form datetime.time
+        * credentials to Amazon Web Service should be stored in connection and
+          indicated by the aws_conn_id parameter
+
+    :type body: dict
+    :param aws_conn_id: The connection ID used to retrieve credentials to
+        Amazon Web Service.
+    :type aws_conn_id: str
+    :param gcp_conn_id: The connection ID used to connect to Google Cloud
+        Platform.
+    :type gcp_conn_id: str
+    :param api_version: API version used (e.g. v1).
+    :type api_version: str
+    :return: Google Storage Transfer service object
+    :rtype: dict
+    """
+    # [START gcp_transfer_job_update_template_fields]
+    template_fields = ('job_name', 'body',)
```

Comment:
```suggestion
    template_fields = ('job_name', 'body','gcp_conn_id', 'aws_conn_id', )
```
You must remember about the changes in the tests.